src/common/formatters.ts

```diff
@@ -2,7 +2,7 @@
 //
 // SPDX-License-Identifier: AGPL-3.0
 
-export const formatDate = (isoDate?: string) => {
+export const formatDate = (isoDate?: string | null) => {
```

Comment:
isoDate already can be undefined. No need for null here

Reply:
@unodgs  
Some values - especially those coming from api - can be `null` so it had to be done this way.
```isoDate?: string === isoDate: string | undefined```